addons/mail/models/mail_channel.py

```diff
@@ -366,46 +364,20 @@ def _notify_compute_recipients(self, message, msg_vals):
         """
         # get values from msg_vals or from message if msg_vals doen't exists
         msg_sudo = message.sudo()
-        mailing_channels = self.filtered('email_send')
         message_type = msg_vals.get('message_type', 'email') if msg_vals else msg_sudo.message_type
         pids = msg_vals.get('partner_ids', []) if msg_vals else msg_sudo.partner_ids.ids
 
         # notify only user input (comment or incoming emails)
         if message_type not in ('comment', 'email'):
             return []
         # notify only mailing lists or if mentioning recipients
-        if not mailing_channels and not pids:
+        if not pids:
             return []
 
         email_from = tools.email_normalize(msg_vals.get('email_from') or msg_sudo.email_from)
         author_id = msg_vals.get('author_id') or msg_sudo.author_id.id
 
         recipients_data = []
-        if mailing_channels:
-            sql_query = """
-                SELECT DISTINCT ON (partner.id) partner.id
-                  FROM res_partner partner
-             LEFT JOIN mail_channel_partner mcp on partner.id = mcp.partner_id
-             LEFT JOIN mail_channel channel on channel.id = mcp.channel_id
-             LEFT JOIN res_users users on partner.id = users.partner_id
-                 WHERE (users.notification_type != 'inbox' or users.id IS NULL)
-                       AND (partner.email != %s or partner.email IS NULL)
-                       AND channel.id = ANY(%s)
-                       AND partner.id != ANY(%s)"""
-            self.env.cr.execute(
-                sql_query,
-                (email_from, mailing_channels.ids, [author_id] if author_id else [], )
-            )
-            for partner_id, in self._cr.fetchall():
-                # ocn_client: will add partners to recipient recipient_data. more ocn notifications. We neeed to filter them maybe
-                recipients_data.append({
-                    'id': partner_id,
-                    'share': True,
-                    'active': True,
-                    'notif': 'email',
-                    'type': 'channel_email',
-                    'groups': []
-                })
         remaining = [pid for pid in pids if pid not in [recipient['id'] for recipient in recipients_data]]
```

Comment:
This line could be removed, we only deals with pids now.

Reply:
Good catch